examples/calibrate_tsqrtP.py

```diff
@@ -0,0 +1,322 @@
+            required to obtain each desired beta in the series.
+            """
+
+            def zonefit(desired):
+                if desired > linear_threshold:
+                    return np.polyval(c_linear[::-1], desired)
+                else:
+                    return np.polyval(
+                        c_nonlinear[::-1], desired - linear_threshold
+                    )
+
+            ret_val = np.vectorize(zonefit)(desired)
+            if ret_val.size > 1:
+                return ret_val
+            else:
+                return ret_val.item()
+
+        fl.next(r"Amplitude*$t_{pulse}$ vs $\beta$", legend=True)
+        t_v_beta.set_plot_color_next()
+        t_v_beta.name("A * $t_{pulse}$")
+        fl.plot(
+            t_v_beta * amplitude,
+            ".",
+            alpha=0.5,
+            label=f"data for {amplitude}",
+        )
+        fl.next(r"Amplitude*$t_{pulse}$ vs $\beta$, zoomed")
+        fl.plot(
+            t_v_beta[r"$\beta$":(None, typical_180)] * amplitude,
+            ".",
+            alpha=0.5,
+            label=f"data for {amplitude}",
+        )
+        # {{{ we extrapolate past the edges of the data using the linear threshold
+        #     to show how the linear is poorly behaved for smaller beta values
+        for_extrap = (
+            psd.nddata(
+                np.linspace(0.5e-6, t_v_beta[r"$\beta$"].max() + 10e-6, 500),
+                r"$\beta$",
+            )
+            .set_units("μs")
+            .set_units(r"$\beta$", "s√W")
+        )
+        for_extrap.copy_props(t_v_beta)
+        fl.plot(
+            for_extrap.eval_poly(c_linear, r"$\beta$")[
+                r"$\beta$":(None, typical_180)
+            ]
+            * amplitude,
+            "--",
+            alpha=0.25,
+            label="linear",
+        )
+        fl.next(r"Amplitude*$t_{pulse}$ vs $\beta$")
+        fl.plot(
+            for_extrap.eval_poly(c_linear, r"$\beta$") * amplitude,
+            "--",
+            alpha=0.25,
+            label="linear",
+        )
+        # }}}
+        full_fit = for_extrap.fromaxis(r"$\beta$").run(prog_plen)
+        fl.plot(full_fit * amplitude, alpha=0.5, label="fit")
+        plt.axvline(
+            x=linear_threshold / 1e-6,  # units of μs
+            alpha=0.3,
+            color=for_extrap.get_plot_color(),
+        )
+        fl.next(r"Amplitude*$t_{pulse}$ vs $\beta$, zoomed")
+        fl.plot(
+            full_fit[r"$\beta$":(None, typical_180)] * amplitude,
+            alpha=0.5,
+            label="fit",
+        )
+        # {{{ Add labels and grids to the fit plots
+        for j in (
+            r"Amplitude*$t_{pulse}$ vs $\beta$",
+            r"Amplitude*$t_{pulse}$ vs $\beta$, zoomed",
+        ):
+            fl.next(j)
+            psd.gridandtick(plt.gca())
+            plt.ylabel(r"$At_{pulse}$ / $\mathrm{\mu s}$")
+            plt.xlabel(r"$\beta$ / $\mathrm{\mu s \sqrt{W}}$")
+        # }}}
```

Comment:
after other PRs are merged, compare to other functions with `switch_to_plot`

Reply:
realized these lines weren't even needed to begin with and just extra garbage so just deleted no need for switch_to_plot